pages/projects/benefits-navigator/index.js

```diff
@@ -11,10 +11,12 @@ import { Collapse } from "../../../components/molecules/Collapse";
 import Image from "next/image";
 import stageDictionary from "../../../lib/utils/stageDictionary";
 import TextRender from "../../../components/text_node_renderer/TextRender";
+import { sortUpdatesByDate } from "../../../lib/utils/sortUpdatesByDate";
 
 export default function BenefitsNavigatorOverview(props) {
   const [pageData] = useState(props.pageData.item);
   const [updatesData] = useState(props.updatesData);
```

Comment:
Let's replace the useState here with the way it's implemented on the other project pages. The way we bring in a lot of the props across the site needs to be made more consistent in general but we'll get to the rest of it later.

Reply:
sounds good! 👍🏾